README.md

```diff
@@ -1,22 +1,19 @@
 # Multi-threaded Google Ban Checker
 
-```
-Usage: main.py [options]
-
-Options:
-  -h, --help            show this help message and exit
-  -d, --debug           Print debug information
-  -n NUM, --threads=NUM
-                        set maximum number of threads to use (default: 100)
-
-  Select one of the following progress option::
-    -%, --show-progress-status
-                        show realtime progress status and a tabular result at
-                        the end
-    -R, --realtime      show results in realtime in csv format
-```
 
-Installation
+**Usage: main.py [options]**
+*Options:*
+
+|   Command    |  |
+| ------------ | ------------ |
+|   -d, --debug   |  Print debug information |
+|  n NUM, --threads  NUM |    set maximum number of threads to use (default: 100)  |
+|  -%, --show-progress-status |    show realtime progress status and a tabular result at the end  |
+|  -R, --realtime |    show results in realtime in csv format  |
+
+
+
+# Installation
 ------------
```

Comment:
We can remove this Horizontal Rule as it's automatically added to us by `#` H1 .